org/lateralgm/file/GmFileReader.java

```diff
@@ -1027,7 +1027,10 @@ private static void readRooms(ProjectFileContext c) throws IOException,GmFormatE
 			rm.put(PRoom.SPEED,in.read4());
 			rm.put(PRoom.PERSISTENT,in.readBool());
 			rm.put(PRoom.BACKGROUND_COLOR,Util.convertGmColor(in.read4()));
-			rm.put(PRoom.DRAW_BACKGROUND_COLOR,in.readBool());
+			//NOTE: Mike Dailly must have done this lol, how does he not know what is conventionally true?
+			int backgroundViewClear = in.read4();
+			rm.put(PRoom.DRAW_BACKGROUND_COLOR,(backgroundViewClear & 1) != 0);
```

Comment:
You seem to be writing twice where you once wrote once, and you're doing so unconditionally. How did this ever work in either mode?

Reply:
You're looking at the reader.

Reply:
Oh, does readBool still read four bytes?

Reply:
Exactly.

Reply:
Ok, you know what, I now see that's the reason why he had the new option negated. If he hadn't done that, then setting the views clear option, would also set the background clear option, if you wanted to take the project back to GM 8.0 instead of GM 8.1.